vehicle-python/tests/mnist_benchmark_setup/vehicle.py

```diff
@@ -0,0 +1,278 @@
+import typing
+from typing import Any, Callable, Dict, Optional, Set, Tuple
+
+import tensorflow as tf
+from command_line_experiment_setup import call_vehicle_to_generate_loss_json
+from utils import internal_error_msg
+
+
+def generate_loss_function(
```

Comment:
These seems to be an entirely copy of the main translation file? Surely we don't want to commit it twice?